src/dto/mod.rs

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+#![allow(missing_docs)]
```

Comment:
Is that temporary only (at the module level at least)? If so I would maybe add a TODO comment or something.

Reply:
Hmm, good question. Should we copy the documentation from `bee-message`? I'll create an issue for now.

Reply:
I actually asked myself the same question. I think that would be the easiest.